third_party/agg/include/agg_renderer_outline_aa.h

```diff
@@ -1371,7 +1371,7 @@ namespace agg
         //---------------------------------------------------------------------
         void profile(const line_profile_aa& prof) { m_profile = &prof; }
         const line_profile_aa& profile() const { return *m_profile; }
-        line_profile_aa& profile() { return *m_profile; }
+        line_profile_aa& profile() { return *(line_profile_aa*)m_profile; }
```

Comment:
Should this not be a const_cast? This looks a bit suspicious to me anyway. Where is this used? m_profile is initialized in the constructor from a const-pointer. It shouldn't be possible to cast away the constness, it hints at a potential problem.